pytorch_mnist/ks_app/app.yaml

```diff
@@ -1,8 +1,11 @@
 apiVersion: 0.3.0
+environments:
+  default:
+    destination:
+      namespace: default
+      server: https://104.154.168.244
+    k8sVersion: v1.8.0
+    path: default
```

Comment:
Thanks for the contribution!  Do we have to modify this file? since that works fine before after updated the `apiVersion` to "0.3.0".

Reply:
Checked, seems that's same with `./examples/github_issue_summarization/ks_app/app.yaml`. I think the change is OK.